snuba/query/configuration.py

```diff
@@ -0,0 +1,218 @@
+from dataclasses import dataclass, field
+from typing import Any
+
+from snuba.state import delete_config as delete_runtime_config
+
+
+class InvalidConfig(Exception):
+    pass
+
+
+@dataclass()
+class Configuration:
```

Comment:
vibe: this is a pretty generic name for something, may be helpful to make it more specific